hands-on-exercise/1-ignite-cli/1-ignitecli.md

```diff
@@ -269,8 +272,39 @@ If you work with a machine using M1 architecture, the Docker image should allow
 3. Install Golang with `brew install go`.
 
 </PanelListItem>
+<PanelListItem number="2">
+
+If you are on Mac M1 with an aggressive anti-virus, it may prevent you from running Ignite, perhaps even within Docker. To circumvent this, you can recreate your Docker image for it to run within Docker **with the Intel CPU emulation**, like so:
```

Comment:
added some comments in Discord

Reply:
https://github.com/ignite/cli/issues/3741

Reply:
Wait, did you not write that Docker without `--platform` also returned an `exit 1`?

Reply:
@xavierlepretre is this resolved?

Reply:
I think my version should still go because M1 in itself is not a blocker for others.

Reply:
yes, without `--platform` it is not worked

Reply:
<img width="346" alt="image" src="https://github.com/cosmos/developer-portal/assets/18665326/2093c0bb-2b01-463a-85a6-798e6d17dee9">